extensions/authentication-passport/src/__tests__/acceptance/fixtures/oauth2-provider.ts

```diff
@@ -0,0 +1,253 @@
+// Copyright IBM Corp. 2018,2019. All Rights Reserved.
```

Comment:
Perhaps use a different term than provider? Provider has a particular meaning and usage in LB4. Just a thought.

Reply:
agreed

Reply:
I am thinking of `MockOauth2LoginApp` or `MockSocialMediaApp`